13-students-doing-data-science.Rmd

```diff
@@ -1 +1,146 @@
+
+### Provide a home base for learners to access resources (and to learn more)
+
+As we discuss in the next section, along with other important factors (such as learners' motivation and having a supportive atmospher), learning strategies can make a difference for learners. Especially when it comes to learning to do data science, there are many tools and resources to keep track of, such as:
+
+- How to download and install R
+- How to download and install R Studio
+- How to install packages
+- How to access resources related to the workshop or course (or simply other resources you wish to share)
+- How to contact the instructor 
+- How to get help and learn more
+
+Having a "home base," where you can remind learners to first look to for resources, can help to lower some of learners' demands in terms of remembering how these tools and resources can be accessed. One way to do this is through a personal website. Another is through GitHub pages. For some organizations, a proprietary learning management system - such as Desire2Learn, Blackboard, Moodle, or Canvas - can be helpful (especially if your learners are accustomed to using them). 
+
+### Draw on other resources
+
+In this section of this chapter, we presented some strategies for teaching data science. There are others that go more into depth on this topic from different perspectives, such as the following: 
+
+- [GAISE Guidelines](https://www.amstat.org/asa/education/Guidelines-for-Assessment-and-Instruction-in-Statistics-Education-Reports.aspx): guidelines for teaching statistics
+- [Teaching Tech Together](http://teachtogether.tech/#the-rules): a book on teaching programming
+- [Data Science for Undergraduates](https://www.nap.edu/catalog/25104/data-science-for-undergraduates-opportunities-and-options): a report on undergraduate data science education
+
+## General strategies related to teaching and learning
+
+The National Academy of Science commissioned a report, [*How People Learn*](https://www.nap.edu/catalog/9853/how-people-learn-brain-mind-experience-and-school-expanded-edition), that aimed to summarize research from the educational psychology and the learning sciences on teaching and learning. In 2018, the book was updated in [*How People Learn II*](https://www.nap.edu/catalog/24783/how-people-learn-ii-learners-contexts-and-cultures), which aimed to emphasize the social and cultural aspects of teaching and learning, which were not as much the focus of the earlier report. In this section, we highlight general strategies related to teaching and learning from the latest report (abbreviated as *HPL2*), with an emphasis on strategies applicable to teaching and learning data science. 
```

Comment:
what I'm not seeing in our resources are pedagogical texts related to classroom instruction - I would argue that K-12 education has largely provided the answers to "How do we teach data science?" question, but yet data science continues to recreate the wheel. 

I would add in resources by Hattie (Visible Learning), Lemov (TLAC 2.0), and potentially Bambrick, along with "Understanding by Design Meets Neuroscience" by McTighe and Willis (the original, Understanding by Design, was done by Wiggins & McTighe)

Reply:
helpful. added as comments and will discuss in meeting so as not to let this slide to editing.